components/signersForm.tsx

```diff
@@ -306,6 +412,23 @@ const SignersForm: FC<{
           errors.proposalDuration = "One value must at least be more than 0";
         }
 
+        if (!!values.bakerAddress) {
+          if (validateAddress(values.bakerAddress) !== ValidationResult.VALID)
+            errors.bakerAddress = `Invalid address ${values.bakerAddress}`;
+          else {
+            try {
+              const account = await fetch(
+                `https://api.ghostnet.tzkt.io/v1/accounts/${values.bakerAddress}`
+              ).then(res => res.json());
+
+              if (account.type !== "delegate" || !account.activationLevel)
```

Comment:
I forgot one thing if there is a `deactivationLevel`, we should show error message like, baker is inactive.

https://ghostnet.tzkt.io/tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb/operations/